FitGoal/FitGoal/Singup&Singin/SignUpViewController.swift

```diff
@@ -131,7 +131,15 @@ class SignUpViewController: UIViewController, AuthenticationTypeSwitcherViewDele
     
     @objc private func presentViewController() {
         if signUpValidator.isUserInputValid() {
-            print("call AuthenticationVC")
+            let customAuth = CustomAuthentication(
+                name: signUpValidator.name.userInput,
+                email: signUpValidator.email.userInput,
+                password: signUpValidator.password.userInput
+            )
+            let vc = AuthenticationViewController(authMethod: .custom(customAuth))
+            vc.modalPresentationStyle = .overCurrentContext
+            vc.modalTransitionStyle = .crossDissolve
```

Comment:
This code here is repeated every time you have to present the view controller. Why don't you move this code to the initialiser of the `AuthenticationViewController`?